src/Routes/Devices/Inventory.js

```diff
@@ -50,6 +50,11 @@ const Inventory = () => {
     setIsRowSelected(isRow);
   };
 
+  const reloadData = () => {
+    fetchDevices();
+    setTimeout(() => setHasModalSubmitted(true), 800);
```

Comment:
We should replace all setTimeout with async/await this way we don't have to set a hard coded wait time.